common/src/main/scala/com/gu/recipeasy/models/CuratedRecipeDB.scala

```diff
@@ -0,0 +1,23 @@
+package com.gu.recipeasy.models
+
+//this model is different from CuratedRecipe as it holds a list of Tag ids
```

Comment:
This seperation isn't particularly clear for me coming at it from having not worked on it. Is it possible to make this clearer in the comment, or better yet in the name of the case classes?